autoload/committia/git.vim

```diff
@@ -197,7 +248,7 @@ function! committia#git#end_of_edit_region_line() abort
         let line = line('$') + 1
     endif
     while line > 1
-        if stridx(getline(line - 1), '#') != 0
+        if getline(line - 1) !~# '^' . s:comment_char()
```

Comment:
This doesn't work if `s:comment_char()` returns `$` because it requires escaping like `\$`. I recommend using `stridx` here.

```suggestion
        if stridx(getline(line - 1), s:comment_char()) != 0
```

Reply:
Good spot, thanks, and the pattern match is no longer needed here as it is again matching on a single character.